Unsupported feature http://xmlpull.org/v1/doc/features.html#indent-output

Hello :)

While trying to perform some tests everything works fine until it tries to generate some reports. I'm constantly getting:

```
> Task :app:gordonMock FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:gordonMock'.
> unsupported feature http://xmlpull.org/v1/doc/features.html#indent-output
```

Good to mention that issue exists only in our project. Your sample on the same configuration works flawless. I have no idea what causes this issue. Creating my fork without [this line](https://github.com/Banno/Gordon/blob/4fb73afc9b9ee318fd67c26c4c96a04d2b69bad4/gordon-plugin/src/main/kotlin/com/banno/gordon/Xml.kt#L10) solves it.

Do you have any ideas or tips where to look in the issue?
Is some dependency conflict possible?

Regards,
Greg

--- a/gordon-plugin/src/main/kotlin/com/banno/gordon/Xml.kt
+++ b/gordon-plugin/src/main/kotlin/com/banno/gordon/Xml.kt
@@ -7,12 +7,24 @@ import java.io.StringWriter
 internal fun xmlDocument(block: XmlSerializer.() -> Unit) = StringWriter().also {
     XmlPullParserFactory.newInstance(System.getProperty(XmlPullParserFactory.PROPERTY_NAME), null).newSerializer().run {
         setOutput(it)
-        setFeature("http://xmlpull.org/v1/doc/features.html#indent-output", true)
+        indentOutput()
         startDocument("UTF-8", null)
         block()
         endDocument()
     }
 }.toString()
+
+private fun XmlSerializer.indentOutput() {
+    try {
+        setFeature("http://xmlpull.org/v1/doc/features.html#indent-output", true)
+    } catch (ignored: Throwable) {
+        try {
+            setProperty("http://xmlpull.org/v1/doc/properties.html#serializer-indentation", "  ")
+            setProperty("http://xmlpull.org/v1/doc/properties.html#serializer-line-separator", "\n")
+        } catch (ignored: Throwable) {
+        }
+    }
+}
 
 internal fun XmlSerializer.attribute(name: String, value: String) {
     attribute("", name, value)

